src/main/java/ee/tuleva/onboarding/account/AccountStatementController.java

```diff
@@ -23,8 +23,8 @@
 
     @ApiOperation(value = "Get pension register account statement")
     @RequestMapping(method = GET, value = "/pension-account-statement")
-    public List<FundBalance> getMyPensionAccountStatement(@ApiIgnore @AuthenticationPrincipal User user,
+    public List<FundBalance> getMyPensionAccountStatement(@ApiIgnore @AuthenticationPrincipal AuthenticatedPerson authenticatedPerson,
                                                           @RequestHeader(value = "x-statistics-identifier", required = false) UUID statisticsIdentifier) {
-        return accountStatementService.getMyPensionAccountStatement(user, statisticsIdentifier);
+        return accountStatementService.getMyPensionAccountStatement(authenticatedPerson.getUser().orElseThrow(RuntimeException::new), statisticsIdentifier);
```

Comment:
Create a new Exception class that you can throw here, instead of using the generic RuntimeException.